src/tasks/harvest_groups.js

```diff
@@ -52,7 +52,7 @@ async function harvest () {
     })
   } catch (error) {
     console.log('Harvest Error:', error)
-    Sentry.captureException(err)
+    Sentry.captureException(error)
```

Comment:
This typo was picked up by VSCode's typescript checker, using `"javascript.implicitProjectConfig.checkJs": true,`